kolibri/plugins/setup_wizard/assets/src/views/importFacility/LoadingTaskPage.vue

```diff
@@ -74,21 +74,18 @@
       },
     },
     beforeMount() {
-      this.startPolling();
+      this.pollTask();
     },
     methods: {
-      startPolling() {
+      pollTask() {
         SetupTasksResource.fetchCollection({ force: true }).then(tasks => {
           this.loadingTask = {
             ...tasks[0],
             facility_name: this.facilityName,
           };
         });
-        if (this.loadingTask.status) {
```

Comment:
This was the wrong condition, it should have said something like "If status is completed/canceled/failed"